### ISSUE TITLE
Support updates when using updatable keys

### ISSUE THREAD

**Issue by maxtoroq** [maxtoroq/DbExtensions#56]
SqlTable uses the new key in the update predicate instead of the old key. Since there's no change tracking, a new overload is required.

A workaround that I'd recommend only for single column keys is the following:

```csharp
SqlBuilder updateSql = db.Table<Entity>()
   .CommandBuilder
   .BuildUpdateStatementForEntity(entity);

updateSql.ParameterValues[updateSql.ParameterValues.Count - 1] = oldId;

db.Execute(updateSql, affect: 1, exact: true);
```

**Comment by maxtoroq** [comment]
Shipped in v6.1.0

**Comment by maxtoroq** [comment]
Added new `SqlTable.Update` overload, e.g.:

```csharp
db.Table<Entity>.Update(entity, oldId);
```
